controllers/spacebindingcleanup/spacebinding_cleanup_controller.go

```diff
@@ -119,7 +111,14 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 }
 
 func (r *Reconciler) isPublicViewerEnabled(ctx context.Context) bool {
-	return r.GetPublicViewerConfig(ctx, r.Client).Enabled()
+	cfg, err := toolchainconfig.GetToolchainConfig(r.Client)
+	if err != nil {
+		log.FromContext(ctx).
+			Error(err, "error retrieving PublicViewerConfig, using default")
+		return toolchainconfig.PublicViewerConfig{}.Enabled()
```

Comment:
As the desired behavior is to have this disabled by default , I'd say that if it's not set we can safely assume it to be disabled

Reply:
Well, in this case it's not because it's not set but you're getting an error while trying to get the `toolchainconfig` resource if I'm seeing it correctly. Thus probably a bigger problem, not related to the publicviewer config not being set.

Reply:
Oh yeah, you're right. I missed that, thanks for pointing it out 👍🏾 I'll fix that!

Reply:
353e5f06b161f2a06da8e3a2f4be292402482c38